schematic/schemas/commands.py

```diff
@@ -114,35 +126,48 @@ def convert(
                 for warning in war:
                     logger.warning(warning)
 
+    if output_path:
+        output_jsonld = output_path
+
+    if output_jsonld is None:
+        output_file_no_ext = re.sub("[.](jsonld|csv|pickle)$", "", schema)
```

Comment:
nit: Try to include full words in the name of variables. I assume `ext` here means `extension`, but I wouldn't have a clue if I am not already looking at how it's created.